src/canisters/frontend/ic-asset/src/batch_upload/plumbing.rs

```diff
@@ -370,7 +370,9 @@ fn content_encoding_descriptive_suffix(content_encoding: &str) -> String {
 // todo: make this configurable https://github.com/dfinity/dx-triage/issues/152
 fn applicable_encoders(media_type: &Mime) -> Vec<ContentEncoder> {
     match (media_type.type_(), media_type.subtype()) {
-        (mime::TEXT, _) | (_, mime::JAVASCRIPT) | (_, mime::HTML) => vec![ContentEncoder::Gzip],
+        (mime::TEXT, _) | (_, mime::JAVASCRIPT) | (_, mime::HTML) => {
+            vec![ContentEncoder::Gzip, ContentEncoder::Brotli]
```

Comment:
Has this been discussed, as a default behavior, at the product level?

I think it would be better to opt-in, configured in .ic-assets.json

Reply:
sns-controlled asset canisters might impossible to sync unless they can opt-in for a subset of files rather than all at once.  This is because a proposal's commit by definition has to be a single call.

Reply:
Will be adjusted once #3792 is merged